File: MasterProduct.DataAccess.SQL/Model/Product.cs
using System;
using System.Collections.Generic;

namespace MasterProduct.DataAccess.SQL.Model
{
    public partial class Product
    {
        public int IdProduct { get; set; }
        public string NameProduct { get; set; }
        public string Properties { get; set; }
        public DateTime ReleaseDate { get; set; }
        public string ManufacturerEmail { get; set; }
        public string IdCountry { get; set; }
        public string Cost { get; set; }
        public string UnitsAvailable { get; set; }
        public string SoldUnits { get; set; }
        public bool StateProduct { get; set; }
        public string ImageProduct { get; set; }
    }
}

File: MasterProduct.UI/ClientApp/app/models/product.ts
export class Product {
    constructor(
        public IdProduct: number,
        public NameProduct: string,
        public Properties: string,
        public ReleaseDate: string,
        public ManufacturerEmail: string,
        public IdCountry: number,
        public Cost: string,
        public UnitsAvailable: string,
        public SoldUnits: string,
        public StateProduct: boolean,
        public ImageProduct: string,
    ) { }
}